source/includes/openstack/_networks.md

```diff
@@ -0,0 +1,111 @@
+### Networks
+
+Networks are privisioned to be able to control data flow to instances. To comminucate with the external network, the network must be connected to a [router]#routers
```

Comment:
Doesn't a link need brackets around the URL? (Talking about the `#routers` part.)